test/cases/linebreak.test

```diff
@@ -0,0 +1,9 @@
+```
+This is a soft
+break and this is a hard\
+break.
+.
+<p>This is a soft
+break and this is a hard<br>
+break.</p>
+```
```

Comment:
Could you remove this and replace it with the canonical cases for this feature in `cases_unimplemented` please 🙏

Reply:
Oh I was looking for the test case but my grep skills were failing me. Will take another look for it.

Reply:
Yeah can do, I was looking for the test case in unimplemented but my grep skills have failed me. Will give it another look.